packages/abstract-interface/src/deployment.rs

```diff
@@ -84,11 +84,14 @@ impl<Chain: CwEnv> Deploy<Chain> for Abstract<Chain> {
         Ok(deployment)
     }
 
-    fn load_from(chain: Chain) -> Result<Self, AbstractInterfaceError> {
-        Ok(Self::new(chain))
+    fn load_from(chain: &mut Chain) -> Result<Self, AbstractInterfaceError> {
+        custom_state(chain);
+        Ok(Self::new(chain.to_owned()))
     }
 }
 
+include!(concat!(env!("OUT_DIR"), "/add_custom_state.rs"));
```

Comment:
This is used to load the right daemon file when building the state. This is always the same, so we could replace that by passing the daemon_state file directly